analyses/response_analysis.py

```diff
@@ -0,0 +1,151 @@
+
+
+import dask.dataframe as dd
+import matplotlib.pyplot as plt
+import pandas as pd
+import seaborn as sns
```

Comment:
If you introduce new dependencies, please maintain the docker scripts. For CI purposes, PaStA needs to be able to run without any issues with the docker image.

Reply:
Sure, I will do that.